huaweicloud/services/ucs/resource_huaweicloud_ucs_policy.go

```diff
@@ -0,0 +1,353 @@
+	"github.com/jmespath/go-jmespath"
+
+	"github.com/chnsz/golangsdk"
+	"github.com/chnsz/golangsdk/pagination"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func ResourcePolicy() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourcePolicyCreate,
+		UpdateContext: resourcePolicyUpdate,
+		ReadContext:   resourcePolicyRead,
+		DeleteContext: resourcePolicyDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: `Specifies the name of the UCS policy.`,
+			},
+			"iam_user_ids": {
+				Type:        schema.TypeList,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				Required:    true,
+				Description: `Specifies the list of iam user IDs to associate to the policy.`,
+			},
+			"type": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: `Specifies the type of the UCS policy.`,
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: `Specifies the description of the UCS policy.`,
+			},
+			"details": {
+				Type:        schema.TypeList,
+				Elem:        PolicyContentSchema(),
+				Optional:    true,
+				Computed:    true,
+				Description: `Specifies the details of the UCS policy.`,
+			},
+			"created_at": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `The created time.`,
+			},
+			"updated_at": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `The updated time.`,
+			},
+		},
+	}
+}
+
+func PolicyContentSchema() *schema.Resource {
+	sc := schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"operations": {
+				Type:        schema.TypeList,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				Optional:    true,
+				Computed:    true,
+				Description: `Specifies the list of operations.`,
+			},
+			"resources": {
+				Type:        schema.TypeList,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				Optional:    true,
+				Computed:    true,
+				Description: `Specifies the list of resources.`,
+			},
+		},
+	}
+	return &sc
+}
+
+func resourcePolicyCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	cfg := meta.(*config.Config)
+	region := cfg.GetRegion(d)
+
+	// createPolicy: Create a UCS Policy.
+	var (
+		createPolicyHttpUrl = "v1/permissions/rules"
+		createPolicyProduct = "ucs"
+	)
+	createPolicyClient, err := cfg.NewServiceClient(createPolicyProduct, region)
```

Comment:
```suggestion
	createPolicyClient, err := cfg.NewServiceClient(createPolicyProduct, "")
```

as the UCS is golbal, we can remove the `region` parameter and create the client with ""

Reply:
fixed